tsconfig.json

```diff
@@ -1,20 +1,9 @@
 // Used by eslint for linting
 {
+	"extends": "./types/tsconfig.json"
```

Comment:
I guess this should have a trailing comma?

I've checked out the branch and tried to do stuff in vscode, and this file as well as the types stuff has errors I don't understand. Something seems to throw off vscode so early that it just says ", expected" (comma) as first thing in the file.

Reply:
I tend to use Webstorm, which doesn't have issues. But I'll check with VSC